src/app/util/af.h

```diff
@@ -130,10 +130,24 @@ extern EmberAfDefinedEndpoint emAfEndpoints[];
 #endif
 
 /**
- * @brief Returns root endpoint of a composed bridged device
+ * @brief Returns parent endpoint for a given endpoint index
  */
 chip::EndpointId emberAfParentEndpointFromIndex(uint16_t index);
 
+/**
+ * @brief Sets the parent endpoint for a given endpoint
+ */
+EmberAfStatus emberAfSetParentEndpointForEndpoint(chip::EndpointId childEndpoint, chip::EndpointId parentEndpoint);
```

Comment:
Please let's not name `emberAf` things that are 100% new to Matter. Suggest just entirely removing  `emberAf` prefix.

Reply:
Noted - I was just trying to stay aligned with the file!